### ISSUE TITLE
Bitbucket repository

### ISSUE THREAD

**Issue by kc9jud** [wlav/cppyy#75]
There seem to be two separate repositories for this project, here and https://bitbucket.org/wlav/cppyy -- they're not in sync, but the issue trackers both seem to be active. Which one is preferred?

**Comment by kc9jud** [comment]
It looks like ef4f5d539d467da9c66e9e9c948289feb9981096 makes this repository primary, but the update to the README never got pushed to Bitbucket?

**Comment by wlav** [comment]
I used to use bitbucket b/c PyPy lived there (they prefer mercurial over git). After they moved to heptapod (b/c bitbucket dropped mercurial support), I moved cppyy to github as apparently everyone is here. However, several of the old issues are still valid, so I wanted to keep them alive for future updates and that's why I didn't drop the repo. And yes, I still respond to new bitbucket issues, too.

Anyway, I'll update the README on bitbucket to send folks to github.
